src/components/ContentBlock.tsx

```diff
@@ -43,7 +43,7 @@ function ContentBlock(props: ContentBlockProps) {
         margin: theme.spacing(2, 0),
       },
 
-      "& div > *": {
+      "& div > :not(div)": {
```

Comment:
I'm pretty sure I screwed something up here, but my lack of foresight tells me to ignore this for now and revisit it later...